README.md

```diff
@@ -1 +1,20 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## A little about me
+
+My name is Kaden Han, I'm a rising third year (that feels weird to say) at UChicago, and I am majoring in **Economics** and **Math**. I am originally from Dallas, Texas.
+
+## Things I like
+
+* Sports
+  * Ultimate
+  * Baseball
+  * Basketball
+  * Football
+  * Tennis
+* Podcasts
+* Running, running the marathon this fall
+* Movies
+* Reading
+* One of my favorite websites is [FiveThirtyEight](https://fivethirtyeight.com)
```

Comment:
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |      x     |              |            |
| **Ease of access**          |   x        |              |            |

Remarks:

* Everything is very clear and concise
* Making a list and adding images was something I struggled with at first because of links not working or messing up the syntax.